test/jdk/java/util/Currency/currency.properties

```diff
@@ -15,7 +15,7 @@ MF=MFF,555,9
 NO=EUR   ,978  ,2,  2099-01-01T00:00:00
 SB=EUR,111,2, 2099-01-01T00:00:00
 US=euR,978,2,2001-01-01T00:00:00
-ZZ	=	ZZZ	,	999	,	3
```

Comment:
This looks weird, but so did the original code. I assume this is to clearly indicate this as a end of list marker.

Reply:
This looks weird indeed. Luckily it's just test code. Did you run the test after this change?

Reply:
All the java/util/Currency tests pass. I also searched the code for "ZZ" but could not find any references in the test.

Reply:
Please do not replace those tabs with spaces as they are intentional for runtime to safely ignore them. I suggest replacing them with Unicode escapes (`\u000b`)

Reply:
`\u000b` is VT   (vertical tab)  
`\u0009` or `\t` perhaps?

Reply:
Right. `\t` is better to avoid such a mistake.

Reply:
Ok, fixed.